frontend/src/utilities/useEnableApplication.tsx

```diff
@@ -91,29 +92,65 @@ export const useEnableApplication = (
   React.useEffect(() => {
     let closed = false;
     if (doEnable) {
-      postValidateIsv(appId, enableValues)
-        .then((response) => {
+      //check the internal route to see if there '/api', if yes, we call the api from internal route, otherwise use previous logic.
+      if (internalRoute && internalRoute.split('/')[1] === 'api') {
+        fetch(internalRoute, {
```

Comment:
can we make this it's own async helper function? There is no need to add more logic here.

Reply:
Changed